### PR TITLE
Fix Migrations for diesel

### PR DESCRIPTION
**Pull request by elliott-davis**
tenor-20242521
Signed-off-by: Travis Elliott Davis <edavis@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-originsrv/src/migrations/2017-12-13-184520_initial_originsrv_migration/up.sql [hunk 1]
```diff
@@ -721,28 +726,6 @@ CREATE OR REPLACE FUNCTION upsert_origin_project_integration_v1 (
     END
 $$ LANGUAGE plpgsql VOLATILE;
 
-CREATE OR REPLACE FUNCTION get_origin_project_integrations_v1 (
```
Comment (chefsalim) [inline components/builder-originsrv/src/migrations/2017-12-13-184520_initial_originsrv_migration/up.sql]: Might be good to add a drop function if exists for these removals.

**Comment by elliott-davis** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/326562304).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/326562304).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**